pyanaconda/bootloader.py

```diff
@@ -2287,12 +2344,12 @@ def write_config_header(self, config):
         header = ("[defaultboot]\n"
                   "defaultauto\n"
                   "prompt=1\n"
-                  "timeout=%(timeout)d\n"
-                  "default=%(default)s\n"
+                  "timeout=%d\n"
                   "target=/boot\n"
-                  % {"timeout": self.timeout,
-                     "default": self.image_label(self.default)})
+                  % self.timeout)
```

Comment:
We are trying to replace `%` everywhere by `.format`. Not required but would be nice :)

Reply:
Okay.